tensorflow-mixin/dashboards/tensorflow-overview.libsonnet

```diff
@@ -80,7 +80,7 @@ local modelRequestRatePanel = {
   },
   targets: [
     prometheus.target(
-      'rate(:tensorflow:serving:request_count{job=~"$job",instance=~"$instance",model_name=~"$model_name"}[$__rate_interval])',
+      'rate(:tensorflow:serving:request_count{' + matcher + '}[$__rate_interval])',
```

Comment:
Since your `matcher` function provides you with `job` and `instance` by default, the `matcher` keyword takes their place perfectly here. However, you still need to manually include `model_name` in the selectors. This applies to any other queries that have selectors other than job/instance

Reply:
```suggestion
      'rate(:tensorflow:serving:request_count{' + matcher + ', model_name=~"$model_name"}[$__rate_interval])',
```